tree/ntuple/v7/test/ntuple_print.cxx

```diff
@@ -0,0 +1,157 @@
+#include <ROOT/RFieldVisitor.hxx>
+#include <ROOT/RNTuple.hxx>
+#include <ROOT/RNTupleModel.hxx>
+
+
+#include <TFile.h>
+
+#include "gtest/gtest.h"
+
+#include <iostream>
+#include <sstream>
+#include <vector>
+
+using RNTupleReader = ROOT::Experimental::RNTupleReader;
+using RNTupleWriter = ROOT::Experimental::RNTupleWriter;
+using RNTupleModel = ROOT::Experimental::RNTupleModel;
+using RFieldBase = ROOT::Experimental::Detail::RFieldBase;
+using RPrintVisitor = ROOT::Experimental::RPrintVisitor;
+using RPrepareVisitor = ROOT::Experimental::RPrepareVisitor;
+template <class T>
+using RField = ROOT::Experimental::RField<T>;
+
+namespace {
+    
+   /**
+   * An RAII wrapper around an open temporary file on disk. It cleans up the guarded file when the wrapper object
+   * goes out of scope.
+   */
+class FileRaii {
+private:
+   std::string fPath;
+   TFile *file = TFile::Open("test.root", "RECREATE");
+public:
+   FileRaii(const std::string &path) : fPath(path) { }
+   FileRaii(const FileRaii&) = delete;
+   FileRaii& operator=(const FileRaii&) = delete;
+   ~FileRaii() {
+      file->Close();
+      std::remove(fPath.c_str());
+   }
+};
+    
+} // anonymous namespace
+
+TEST(RNtuplePrint, FullString)
+{
+   FileRaii fileGuard("test.root");
+   {
+      auto model = RNTupleModel::Create();
+      auto fieldPt = model->MakeField<float>("pt");
+      auto ntuple = RNTupleWriter::Recreate(std::move(model), "Staff", "test.root");
+      *fieldPt = 5.0f;
+      ntuple->Fill();
+   }
+   auto ntuple2 = RNTupleReader::Open("Staff", "test.root");
+   std::ostringstream os;
+   ntuple2->PrintInfo(os);
+   std::string fString{"************************************ NTUPLE ************************************\n* N-Tuple  : Staff                                                             *\n* Entries : 1                                                                  *\n********************************************************************************\n* Field 1   : pt (float)                                                       *\n********************************************************************************\n"};
```

Comment:
I would format the expected string according to the line breaks of its contents (e.g. break before `N-Tuple` in the first line)